src/members/Member.css

```diff
@@ -1,50 +1,57 @@
 .container-members {
-    background: #f7f9fa;
-    height: auto;
-    width: 90vw;
-    margin: 5em auto;
-    -webkit-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
-    -moz-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
-    box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
-  }
-  
+  background: #f7f9fa;
+  height: auto;
+  width: 90vw;
+  margin: 5em auto;
+  -webkit-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
+  -moz-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
+  box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
+}
+
 label {
-    color: #24b9b6;
-    font-size: 1.2em;
-    font-weight: 400;
-  }
-  
-  h1 {
-    color: #24b9b6;
-    padding-top: 0.5em;
-  }
-  
-  .error {
-    color: red;
-  }
-  .error-message {
-    color: #ff6565;
-    padding: 0.5em 0.2em;
-    height: 1em;
-    position: absolute;
-    font-size: 0.8em;
-  }
-  
-  .field {
-    border: 1px solid #1c6ea4;
-    width: 100%;
-    height: 2em;
-  }
-  
-  .button-group {
-    width: 100%;
-  }
-  
-  .membersList {
-    width: 85vw;
-    text-align: center;
-    padding: 2em;
-    margin: 3em 3em;
-    padding: 2em;
-  }
-  
\ No newline at end of file
+  color: #24b9b6;
+  font-size: 1.2em;
+  font-weight: 400;
+}
+
+h1 {
+  color: #24b9b6;
+  padding-top: 0.5em;
+}
+
+.error {
+  color: red;
+}
+.error-message {
+  color: #ff6565;
+  padding: 0.5em 0.2em;
+  height: 1em;
+  position: absolute;
+  font-size: 0.8em;
+}
+
+.field {
+  border: 1px solid #1c6ea4;
+  width: 100%;
+  height: 2em;
+}
+
+.button-group {
+  width: 100%;
+}
+
+.membersList {
+  width: 85vw;
+  text-align: center;
+  padding: 2em;
+  margin: 3em 3em;
+  padding: 2em;
+}
+
+@media only screen and (max-width: 644px) {
+  .container-members {
+    overflow: hidden;
```

Comment:
Why this line? 

P.S:- Probably we can add `overflow:hidden` directly to the class without using media query :thinking:

Reply:
ya @devkapilbansal , we can but i think horizontal scroll bar on full screen is not a good option because there is nothing to scroll , main problem starts when screen size get smaller or less then 644px .
@keshakaneria @codesankalp any suggestions ??

Reply:
@naveen8801 I agree! We dont need the horizontal scroll bar for screen size greater than 644px. So it fine i suppose.

Reply:
@keshakaneria  Won't `overflow:auto` be better than `hidden` ?

Reply:
Umm.. yes I suppose since max-width is given for smaller screens, auto works more better than hidden. @naveen8801 you can use `overflow: auto;` in place of `hidden`.

Reply:
> Umm.. yes I suppose since max-width is given for smaller screens, auto works more better than hidden. @naveen8801 you can use `overflow: auto;` in place of `hidden`.

sure @keshakaneria  👍